sdk/python/examples/jobset.py

```diff
@@ -0,0 +1,43 @@
+import jobset
+from jobset.models.jobset_v1alpha2_job_set import JobsetV1alpha2JobSet
+from kubernetes.client.models.v1_job_template_spec import V1JobTemplateSpec
+
+jobset_example = jobset.models.jobset_v1alpha2_job_set.JobsetV1alpha2JobSet(
+    api_version="0",
+    kind="0",
+    metadata=None,
+    spec=jobset.models.jobset_v1alpha2_job_set_spec.JobsetV1alpha2JobSetSpec(
+        failure_policy=jobset.models.jobset_v1alpha2_failure_policy.JobsetV1alpha2FailurePolicy(
+            max_restarts=56,
+        ),
+        replicated_jobs=[
+            jobset.models.jobset_v1alpha2_replicated_job.JobsetV1alpha2ReplicatedJob(
+                name="0",
+                network=jobset.models.jobset_v1alpha2_network.JobsetV1alpha2Network(
+                    enable_dns_hostnames=True,
+                ),
+                replicas=56,
+                template=V1JobTemplateSpec(),
+            )
+        ],
+        success_policy=jobset.models.jobset_v1alpha2_success_policy.JobsetV1alpha2SuccessPolicy(
+            operator="0",
+            target_replicated_jobs=["0"],
+        ),
+        suspend=True,
+    ),
+    status=jobset.models.jobset_v1alpha2_job_set_status.JobsetV1alpha2JobSetStatus(
+        replicated_jobs_status=[
+            jobset.models.jobset_v1alpha2_replicated_job_status.JobsetV1alpha2ReplicatedJobStatus(
+                failed=56,
+                name="0",
+                ready=56,
+                succeeded=56,
+            )
+        ],
+        conditions=[None],
+        restarts=56,
+    ),
+)
+
+print(jobset_example)
```

Comment:
Can we use the [dynamic client](https://github.com/kubernetes-client/python-base/blob/master/dynamic/client.py) to create this JobSet?

Reply:
I'm not really sure about dynamic client.  I found that you can do `create_namespaced_custom_object` which will allow creation of a CRD assuming that the CRD is installed in your cluster.  

Is this sufficient?  Dynamic client seemed a bit more complicated than what I think we need.

Reply:
I see, can you update the example to use this function and then we can manually verify that the example works? The goal is to have some copy-pastable example to create a simple JobSet on a cluster.

Reply:
> I see, can you update the example to use this function and then we can manually verify that the example work (i.e. I will try running the script and confirming a JobSet is created in my cluster)?

You can run `python3 examples/create_jobset.py` and that will create it.  

> The goal is to have some copy-pastable example to create a simple JobSet on a cluster, and perhaps does simple CRUD Operations like listing jobsets, updating a jobset, etc. For this I think the dynamic client is necessary, is it not?


lol, tbh I only learned about the python kubernetes client over the weekend.  I see that you can use CRUD operations without a dynamic client (https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/CustomObjectsApi.md).

We have a CI fix to make this workable but not sure how to get docker installed in the runner.  One option is to switch the runner to use the same runner we use for e2e tests.  Or I could go about installing docker on the CI if we want to keep the same image.

Reply:
I don't really mind which image we use the same image as we use for e2e tests so that we can run unit tests in docker. I think it would be cleaner than installing docker locally in the script, what do you think @ahg-g?

Reply:
I think for simply creating a jobset using the customobjectapi is fine, so the example in this PR is good. But for cases where the user is building an operator that requires watching on JobSet objects, then the dynamic client is the better option. It would be nice if we can add another example that uses the dynamic client to create and create a watch, but not urgent.

Reply:
https://github.com/kubernetes/test-infra/pull/29947 is a PR to change the image.  Hopefully it works..

Reply:
I tried to ask in sig-testing why is the docker daemon not started on this runner but no response.  I'm not sure of the path forward if I can't assume that docker is capable of running containers in the CI.